python/tutorials/03-matrix-multiplication.py

```diff
@@ -397,6 +397,7 @@ def benchmark(M, N, K, provider):
         global verbose
         if verbose:
             print(f'SIZE: {M},{N},{K}   Best tuning config: ({matmul_kernel.get_best_config()})')
+            print(f'SIZE: {M},{N},{K} TIME: {ms:.3f} ms, {min_ms:.3f} min_ms, {max_ms:.3f} max_ms')
```

Comment:
I assume this is for debugging purpose. If so, let's restore the tutorial.

Reply:
I feel it is useful here as it is convenient for us to not only just compare against best tunning configuration, but also provide the timing to double confirm it is the best configuration. The TFLOPS info are printed regardless of command line options, so I only added wall time information here.